hydrolib/core/io/polyfile/model.py

```diff
@@ -0,0 +1,28 @@
+from hydrolib.core.io.polyfile.components import PolyObject
+from hydrolib.core.io.polyfile.serializer import write_polyfile
+from hydrolib.core.io.polyfile.parser import read_polyfile
+from hydrolib.core.basemodel import FileModel
+from typing import Callable, Sequence
+
+
+class PolyFile(FileModel):
+    """Poly-file (.pol/.pli/.pliz) representation."""
+
+    has_z_values: bool = False
+    objects: Sequence[PolyObject] = []
```

Comment:
It can technically be either a polygon or a line, I wasn't sure what other name to use than object, but I'd be happy to change it if you have a better suggestion.